tests/scheduler/test_models.py

```diff
@@ -33,8 +34,9 @@ def assert_shift_conflict_count(
     shift,
     hard_conflict_count,
     soft_conflict_count,
-    grace=timedelta(hours=1),
+    *args,
 ):
+    grace = args[0] if args else timedelta(hours=1)
```

Comment:
```suggestion
    grace,
):
    grace = grace if grace is not None else timedelta(hours=1)
```

as in [scheduler/signals.py](https://github.com/coders4help/volunteer_planner/pull/612/files#diff-8f845bde33120acc737b16f82259b3722d893a9dbefed2642bdc9a13d3e2a83e)?

Reply:
thats doesn't work, because `None` is actually a valid value (it was like this before, but pytest said "no!")

Reply:
Gosh - you're right. 🙈 
Thanks for the re-review.